public/v1/apps/elasticsearch.json

```diff
@@ -0,0 +1,83 @@
+{
+    "captainVersion": 1,
+    "documentation": "Elasticsearch (Single and Cluster). Taken from docker.elastic.co/elasticsearch/elasticsearch",
+    "dockerCompose": {
+        "version": "3.3",
+        "services": {
+            "$$cap_appname": {
+                "image" : "docker.elastic.co/elasticsearch/elasticsearch:$$cap_elasticsearch_version",
+                "notExposeAsWebApp": "true",
+                "volumes": [
+                    "$$cap_appname-elasticsearch-data:/usr/share/elasticsearch/data"
+                ],
+                "restart": "always",
+                "environment": {
+                    "ES_JAVA_OPTS": "-Xms512m -Xmx512m",
+                    "cluster.name": "$$cap_elasticsearch_cluster_name",
+                    "http.port": "$$cap_container_port",
+                    "node.master": "$$cap_elasticsearch_node_master",
+                    "node.data": "$$cap_elasticsearch_node_data",
+                    "discovery.zen.minimum_master_nodes": "$$cap_elasticsearch_minimum_master_nodes",
+                    "discovery.zen.ping.unicast.hosts": "$$cap_elasticsearch_unicast_hosts"
+                },
+                "ports": [
+                    "$$cap_host_port:$$cap_container_port"
```

Comment:
You shouldn't need to port forward. As far as I know, port 9200 of Elastic Search is just an HTTP port. In that case:
- Remove `notExposeAsWebApp`
- Remove `cap_host_port`
- Set  `containerHttpPort` to `$$cap_container_port`

Reply:
makes sense.